scripts/update-built-docs.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+set -e
+
+cleanup() {
+    git ls-files ./ | xargs -r -n 1 rm
+    rm -rfd ./*
+}
+
+copyToTemp() {
+    TEMP_DIR="$(mktemp -d --suffix=_doctum-build-php-webdriver)"
+    cp -rp build/dist/* "${TEMP_DIR}"
+    cp ./scripts/docs-template.html "${TEMP_DIR}/index.html"
+}
+
+emptyAndRemoveTemp() {
+    mv "${TEMP_DIR}"/* ./
+    # Create symlink for main to latest
+    ln -s -r ./main ./latest
+    # Create symlink for main to master
+    ln -s -r ./main ./master
+    # Create symlink for main to community
+    ln -s -r ./main ./community
+    rm -rf "${TEMP_DIR}"
+}
+
+commitAndPushChanges() {
+    # Push the changes, only if there is changes
+    git add -A
+    git diff-index --quiet HEAD || git commit -m "Api documentations update ($(date --utc))" -m "#apidocs" && if [ -z "${SKIP_PUSH}" ]; then git push; fi
```

Comment:
```suggestion
    git diff-index --quiet HEAD || git commit -m "Api documentations update ($(date --rfc-3339=seconds --utc))" -m "#apidocs" && if [ -z "${SKIP_PUSH}" ]; then git push; fi
```

To be sure its not a locale-specific date.
```sh
$ date --utc
Čt 22. dubna 2021, 11:32:49 UTC
$ date --rfc-3339=seconds --utc
2021-04-22 11:32:53+00:00
```

Reply:
Nice, I will apply that